### ISSUE TITLE
Request for npm publish

### ISSUE THREAD

**Issue by magnusarinell** [konsumer/gatsby-remark-graph#5]
Hi,

Could you please do an npm publish? I need the PR from @ajrussellaudio. I've been using a local clone of this repository but that's no longer feasible.

Thanks!

**Comment by ajrussellaudio** [comment]
If it's mission critical, you can install straight from this Github repo:

```
npm install git+ssh://git@github.com:konsumer/gatsby-remark-graph.git
```

Or if you need HTTPS:

```
npm install git+https://github.com/konsumer/gatsby-remark-graph.git
```

**Comment by konsumer** [comment]
Published at `v0.2.2`
